addons/crm_sms/views/crm_lead_views.xml

```diff
@@ -26,4 +26,26 @@
             'default_res_id': active_id,
         }"
     />
-</odoo>
\ No newline at end of file
+
+    <record id="crm_case_tree_view_my_activities_extend" model="ir.ui.view">
```

Comment:
Inherited view should be named like an original one, as the module will be added as prefix for the full xml id.

Example: id="crm_lead_view_list_activities", full xml id is crm_sms.crm_lead_view_list_activities -> we know this is an inherit. Therefore to keep xml ids "short" we just use the same guidelines for naming inherited xml ids.